Makefile

```diff
@@ -1,4 +1,11 @@
+NAMESPACE := odigos-system
+CONFIGMAP_NAME := odigos-deployment
+
 ORG := keyval
+
+# Cluster Version
+TAG := $(shell kubectl get configmap $(CONFIGMAP_NAME) -n $(NAMESPACE) -o jsonpath='{.data.ODIGOS_VERSION}')
```

Comment:
Also possible is `TAG ?= $(shell odigos version --short)`

Eden has a similar change in the Makefile at: https://github.com/odigos-io/odigos/pull/1250/files

Reply:
You are right!
In this PR I also added a flag for cli version and cluster version.
So, I'll use it :)